### PR TITLE
Feature/more emojis

### PR DESCRIPTION
**Pull request by Mr1BitL8r**
Added more emojis for the English template and introduced the idea of using unique names that match the names used on https://emojipedia.org/halloween/ so that it is easier to identify, add and later also add more translations.

### CODE REVIEW

**Comment by tmttn** [comment]
Whoa! 😮

File: lang/en.js [hunk 1]
```diff
@@ -1,42 +1,422 @@
...
+    },
+    {
+      name: 'alienmonster',
+      uniqueName: 'Alien Monster',
+      emoji: '👾',
+    },
+    {
+      name: 'heart',
+      uniqueName: 'Anatomical Heart',
+      emoji: '🫀',
```
Comment (illusi0nary) [inline lang/en.js]: This seems to be correct - is that only me ? 
I just see 🫀 - that might cause problems?

File: lang/en.js [hunk 2]
```diff
@@ -1,42 +1,422 @@
...
+    },
+    {
+      name: 'owl',
+      uniqueName: 'Owl',
+      emoji: '🦉',
+    },
+    {
+      name: 'disguises',
+      uniqueName: 'Performing Arts',
+      emoji: '🥸',
```
Comment (illusi0nary) [inline lang/en.js]: Same here it is only showing 🫀

**Review by illusi0nary — CHANGES_REQUESTED** [review]

File: lang/en.js (hunk 2 above)
Comment (Mr1BitL8r) [reply lang/en.js]: The "Performing Arts" one actually was a copy-paste issue and should be fixed now. :)

File: lang/en.js (hunk 1 above)
Comment (Mr1BitL8r) [reply lang/en.js]: Thanks for the hint. Some emojis are too new (e.g. "headstone" as well) so I changed them into comments and added the hint that they arrive later this year wit Unicode/Emoji 13.0, e.g. for Android 11.

**Review by illusi0nary — APPROVED** [review]
All good :-)

**Review by LucasCtrl — APPROVED** [review]
I'm going to merge the changes but I'll delete the unique ID part and the comment one.
I the future, we are maybe going to change the file structure

**Pull request merged**